### PR TITLE
data:ShyamTawli

### PR DESCRIPTION
**Pull request by shyamtawli**

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Your filename is missing `.json` at the end. Without this, your profile won't be added properly.

**Comment by shyamtawli** [comment]
@Panquesito7  how can i update on current branch ?

**Comment by Aadarsh805** [comment]
@shyamtawli your username must match the file name, make it `shyamtawli.json`

File: data/shyamtawli.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend Developer | HTML | CSS | ReactJS | DSA-Java |",
+  "avatar": "https://github.com/SaraJaoude.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/shyam_tawli",
+      "icon": "FaTwitter"
+    }
```
Comment (Panquesito7) [inline data/shyamtawli.json:12]: Missing comma.
```suggestion
    },
```

File: data/shyamtawli.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "GitHub",
+      "url": "https://github.com/shyamtawli",
+      "icon": "FaGithub"
+    }
```
Comment (Panquesito7) [inline data/shyamtawli.json:17]:
```suggestion
    },
```

File: data/shyamtawli.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    {
+      "name": "Instagram",
+      "url": "https://instagram.com/shyamtawli",
+      "icon": "FaInstagram"
+    }
```
Comment (Panquesito7) [inline data/shyamtawli.json:22]:
```suggestion
    },
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍  Great collaboration 💪

**Pull request merged**